homeassistant/components/plugwise/gateway.py

```diff
@@ -64,95 +66,99 @@ async def async_setup_entry_gw(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     try:
         connected = await api.connect()
-
         if not connected:
-            _LOGGER.error("Unable to connect to Smile")
+            _LOGGER.error("Unable to connect to the Smile/Stretch")
             raise ConfigEntryNotReady
-
     except InvalidAuthentication:
         _LOGGER.error("Invalid username or Smile ID")
         return False
-
     except PlugwiseException as err:
-        _LOGGER.error("Error while communicating to device %s", api.smile_name)
+        _LOGGER.error("Error while communicating to the Smile/Stretch")
```

Comment:
I think it would help to at least communicate which device this error is for. If you have multiple devices, the logs now won't tell you what is wrong.